### DISCUSSION TITLE
Downloaded MD location?

### DISCUSSION THREAD

**Discussion by in8dna** [Hugo-COLLIN/SaveMyPhind-conversation-exporter#111] [Issues & Questions] [answered]
I'm looking for the downloaded MD file. Hints where it can be found after clicking on the extension icon?

**Comment by Hugo-COLLIN** [comment, marked as answer]
Hey @in8dna,
The md file should be in your download folder. If not, could you give more details, like the page you want to download or any error appearing in the developer console ?
